src/loaders/eventLoader.js

```diff
@@ -225,7 +226,8 @@ const getFilterOptionNames = async (filters, headers) => {
 };
 
 // Empty filter sometimes returned for saved maps
-const isValidDimension = ({ dimension, items }) =>
-    Boolean(dimension && (!items || items.length));
+// Dimension without filter and empty items array returns false
+const isValidDimension = ({ dimension, filter, items }) =>
+    Boolean(dimension && (filter || (!items || items.length)));
```

Comment:
Minor - The extra parenthesis shouldn't be required here - just `(filter || !items || items.length)`